### PR TITLE
BINF-481: implement to prod

### PR DESCRIPTION
**Pull request by Shenglai**

### CODE REVIEW

File: samtools_mpileup_tool/multi_samtools_mpileup.py [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+        raise ValueError("Exceptions raised during processing.")
+
+    # Check and merge outputs
+    p = pathlib.Path('.')
+    outputs = list(p.glob("*.mpileup"))
+
+    # Sanity check
+    if len(run_commands) != len(outputs):
+        logger.error("Number of output files not expected")
+
```
Comment (czyszCTDS) [inline samtools_mpileup_tool/multi_samtools_mpileup.py:214]: Is it necessary to merge the output files here?

**Review by czyszCTDS — CHANGES_REQUESTED** [review]
Awesome work!

However, there do need to be tests for CI/CD to be successful, could you copy some tests in and update them as needed so that `make test` passes locally?

File: samtools_mpileup_tool/multi_samtools_mpileup.py (hunk 1 above)
Comment (Shenglai) [reply samtools_mpileup_tool/multi_samtools_mpileup.py:214]: no. the varscan/somaticsniper will take a list of mpileup as inputs. so just need to check the number of outputs. 👍

**Comment by Shenglai** [comment]
> Awesome work!
> 
> However, there do need to be tests for CI/CD to be successful, could you copy some tests in and update them as needed so that `make test` passes locally?

Done. just realized that i forgot to push.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+COPY ./dist/ /opt
+WORKDIR /opt
+
+RUN apt-get update \
+	&& apt-get install make \
+	&& rm -rf /var/lib/apt/lists/*
+
+RUN make init-pip \
+  && ln -s /opt/bin/${BINARY} /usr/local/bin/${BINARY}
```
Comment (czyszCTDS) [inline Dockerfile:20]:
```suggestion
  && ln -s /opt/bin/${BINARY} /bin/${BINARY}
```
This should fix the current build issue

**Review by czyszCTDS — APPROVED** [review]

**Pull request merged**